applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -13,7 +13,7 @@
     </div>
     <div class="newsletter-card__wrapper">
         @emailListings.zipWithRowInfo.map { case (emailListing, row) =>
-            <div class="newsletter-card" data-component="newsletter-card @emailListing.name">
+            <div class="newsletter-card" data-component="newsletter-card @emailListing.identityName">
```

Comment:
One slightly "safer" alternative you could use is `emailListing.identityNameOverride.getOrElse(emailListing.name)`.

I think the approach you have is the one we should use, but this one would leave the tracking of most of the newsletters unchanged.